mmsoap/client.py

```diff
@@ -72,58 +55,77 @@ def create(self, object_name):
         return self.client.factory.create(object_name)
 
     def set_userId(self, userId):
+        """ Set user id on the authentication element. """
         self.authentication.userId = userId
 
     def set_password(self, password):
+        """ Set password on the authentication element. """
         self.authentication.password = password
 
     def check_user(self):
-        return self.client.service.checkUser(self.authentication)\
-            .accountDetails
-
-    def send_messages(self, recipients, content, send_mode='normal'):
         """
-        Send a message to `recipients`.
+        Check user account details.
 
-        Arguments:
-        recipients -- iterable of recipient numbers
-        content -- message content
+        :return: The account details including used and remaining credit limits.
+        """
+        return self.client.service.checkUser(self.authentication).accountDetails
 
-        Keyword arguments:
-        send_mode -- used for testing. One of:
-            * normal (default) -- send as normal
-            * dropAll -- drop (not send) the requested messages, return
-              random errors and successes
-            * dropAllWithErrors -- drop the requested messages,
-              return errors
-            * dropAllWithSuccess -- drop the requested messages,
-              return success
+    def send_messages(self, recipients, content, send_mode='normal', scheduled=None, report=None, seq=None):
```

Comment:
It should be a way to specify message id (uid) for scheduled message. Otherwise we can't unschedule it. Or at least uid=0 should be used when unscheduling message.